src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/PersonServiceIntTest.kt

```diff
@@ -21,16 +22,6 @@ class PersonServiceIntTest : IntegrationTestBase() {
   @Autowired
   lateinit var personService: PersonService
 
-  @Test
-  fun `should not log record update when no change in matching fields`() {
-    val person = createRandomProbationPersonDetails()
-    val existingPerson = createPersonWithNewKey(person)
-
-    personService.processPerson(person) { existingPerson }
-
-    checkEventLogExist(existingPerson.crn!!, CPRLogEvents.CPR_RECORD_UPDATED, times = 0)
-  }
```

Comment:
covered by the other test modified : `should not log record update when no change in matching fields but different order`